packages/contracts-bedrock/test/L1/StandardValidator.t.sol

```diff
@@ -106,911 +104,622 @@ contract StandardValidatorTest is CommonTest {
-        vm.mockCall(address(_asr), abi.encodeCall(ISemver.version, ()), abi.encode("1.0.0"));
-        assertEq("PDDG-ANCHORP-10,PLDG-ANCHORP-10", validate(true));
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         AnchorStateRegistry version is invalid.
+    function test_validate_anchorStateRegistryInvalidVersion_succeeds() public {
+        vm.mockCall(address(anchorStateRegistry), abi.encodeCall(ISemver.version, ()), abi.encode("0.0.1"));
+        assertEq("PDDG-ANCHORP-10,PLDG-ANCHORP-10", _validate(true));
+    }
 
-        // Test invalid anchor state registry factory
-        _mockValidationCalls();
-        vm.mockCall(
-            address(_asr), abi.encodeCall(IAnchorStateRegistry.disputeGameFactory, ()), abi.encode(address(0xbad))
-        );
-        assertEq("PDDG-ANCHORP-30,PLDG-ANCHORP-30", validate(true));
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         DelayedWETH version is invalid.
+    function test_validate_delayedWETHInvalidVersion_succeeds() public {
+        vm.mockCall(address(delayedWeth), abi.encodeCall(ISemver.version, ()), abi.encode("0.0.1"));
+
+        // One last mess here, during local tests delayedWeth refers to the contract attached to
+        // the FaultDisputeGame, but during fork tests it refers to the one attached to the
+        // PermissionedDisputeGame. We'll just branch based on the test type.
+        if (isForkTest()) {
+            assertEq("PDDG-DWETH-10", _validate(true));
+        } else {
+            assertEq("PLDG-DWETH-10", _validate(true));
+        }
+    }
 
-        // Test invalid DelayedWETH version
-        _mockValidationCalls();
-        vm.mockCall(address(_weth), abi.encodeCall(ISemver.version, ()), abi.encode("1.0.0"));
-        assertEq(string.concat(errorPrefix, "-DWETH-10"), validate(true));
+    /// @notice Tests that the validate function successfully returns the right error when the
+    ///         PreimageOracle version is invalid.
+    function test_validate_preimageOracleInvalidVersion_succeeds() public {
+        vm.mockCall(address(preimageOracle), abi.encodeCall(ISemver.version, ()), abi.encode("0.0.1"));
+        assertEq("PDDG-PIMGO-10,PLDG-PIMGO-10", _validate(true));
+    }
 
-        // Test invalid DelayedWETH implementation for permissioned game
-        _mockValidationCalls();
-        vm.mockCall(
-            address(proxyAdmin),
-            abi.encodeCall(IProxyAdmin.getProxyImplementation, (address(_weth))),
-            abi.encode(address(0xbad))
-        );
-        assertEq(string.concat(errorPrefix, "-DWETH-20"), validate(true));
-
-        // Test invalid DelayedWETH owner
-        _mockValidationCalls();
-        vm.mockCall(address(_weth), abi.encodeCall(IDelayedWETH.proxyAdminOwner, ()), abi.encode(address(0xbad)));
-        assertEq(string.concat(errorPrefix, "-DWETH-30"), validate(true));
```

Comment:
Comment only: 
not explicitly tested, but this error appears in `test_validate_invalidProxyAdminOwner_succeeds`, so its fine.